tests/platform_tests/test_intf_fec.py

```diff
@@ -153,14 +160,15 @@ def skip_ber_counters_test(intf_status: dict) -> bool:
         if speed not in SUPPORTED_SPEEDS:
             continue
 
-        fec_corr = intf.get('fec_corr', '').lower()
-        fec_uncorr = intf.get('fec_uncorr', '').lower()
-        fec_symbol_err = intf.get('fec_symbol_err', '').lower()
+        # Removes commas from "show interfaces counters fec-stats" (i.e. 12,354 --> 12354) to allow int conversion
+        fec_corr = intf.get('fec_corr', '').replace(',', '').lower()
+        fec_uncorr = intf.get('fec_uncorr', '').replace(',', '').lower()
+        fec_symbol_err = intf.get('fec_symbol_err', '').replace(',', '').lower()
         # Check if fec_corr, fec_uncorr, and fec_symbol_err are valid integers
```

Comment:
This looks like a mistake in conflict handling, the replace comma should stay in the try block.